nettacker/core/module.py

```diff
@@ -39,6 +39,9 @@ def __init__(
                 self.module_inputs[module_extra_args] = self.module_inputs["modules_extra_args"][
                     module_extra_args
                 ]
+        if options.user_wordlist:
+            self.module_inputs["read_from_file"] = options.user_wordlist
```

Comment:
unnecessary copy - just use 'read_from_file' - no need to create an additional user_wordlist, only to copy it into 'read_from_file'